.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
```

Comment:
Is this the same as "filter based on tag"?

Reply:
no this is us manually creating a "release" in github.
https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository